.idea/misc.xml

```diff
@@ -1,4 +1,3 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this an accident?

Reply:
Yes it was, thank you for calling it out!